cmd/jobbuzz-api/main.go

```diff
@@ -8,5 +8,8 @@ import (
 func main() {
 	r := gin.Default()
 	r.GET("/ping", controller.Ping)
+	r.GET("/scrape-jobcenter", controller.ScrapeJobcenter)
+	r.GET("/scrape-bruneida", controller.ScrapeBruneida)
```

Comment:
should add a new `cmd` for the scraper

Reply:
I put the actual scraper in pkg, and the controller uses the scraper

Reply:
Yes, what I mean is that we shouldn't allow the scraper to be triggered using http requests, it should be an executable program instead.

So moving it to something like `cmd/jobalert-scraper/main.go` and having the scraper run in `func main()` is probably better.